targets/bmv2/test/ptf_protobuf_base.py

```diff
@@ -26,10 +26,25 @@ def tearDown(self) -> None:
 
     def createWriteRequest(self) -> p4runtime_pb2.WriteRequest:
         req = p4runtime_pb2.WriteRequest()
+        election_id = req.election_id
+        election_id.high = 0
+        election_id.low = 1
         req.device_id = self.device_id
         return req
 
-    def setupCtrlPlane(self) -> None:
+    def _write(self, req):
```

Comment:
You can use `self.write_request` from the base test instead of reimplementing these functions here.